src/main/java/org/jboss/logmanager/formatters/TextBannerFormatter.java

```diff
@@ -37,7 +37,7 @@ public TextBannerFormatter(final Supplier<String> bannerSupplier, final ExtForma
     public String getHead(final Handler h) {
         final String dh = Objects.requireNonNullElse(delegate.getHead(h), "");
         final String banner = Objects.requireNonNullElse(bannerSupplier.get(), "");
-        return banner + dh;
+        return banner.concat(dh);
```

Comment:
Could you add a comment here about why we are using `concat` instead of `+`? I'm sure I'll forget and then decide to "optimize" this later otherwise. ;-)